src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -827,3 +827,15 @@
     text: >
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
+
+helps['acr check-health'] = """
+type: command
+short-summary: Gets health information on the environment and, optionally, a target registry.
```

Comment:
nit: too many comas
`Gets health information on the environment and optionally a target registry.` should be good.

Reply:
Also, do we want to add an example where we don't specify target registry?

Reply:
Agreed, and added an example without a registry.